### ISSUE TITLE
about M-bias

### ISSUE THREAD

**Issue by feihongloveworld** [dpryan79/MethylDackel#147]
hi sir:
        there are M-bias in the ends of reads due to the end repairing. so are there any parameters to cut those end bases of reads?

**Comment by dpryan79** [comment]
Yes, note the `--OT 0,0,2,8` or whatever suggested at the bottom right of the images. Those are suggested options for use with methylation extraction.
